client/app/optionsets/landuse-form.js

```diff
@@ -118,6 +118,17 @@ export const DCPONLYCHANGETHEELIMINATIONOFAMAPPEDBUTUNIMP = {
   },
 };
 
+export const DCPYESMAPPEDBUTUNIMPROVEDSTREETELIMATED = {
```

Comment:
When I look up the metadata for this field I'm getting that it's a boolean -- in that case, there's no need to add an extra optionset here, instead we use a "common" reusable optoinset that you'll see in my comment on optionset.js file.

https://dcppfsuat2.crm9.dynamics.com/api/data/v9.1/EntityDefinitions(LogicalName=%27dcp_landuse%27)/Attributes(LogicalName=%27dcp_yesmappedbutunimprovedstreetelimated%27)